README.md

```diff
@@ -92,6 +94,25 @@ The following files are the central input files for ABACUS. Before executing the
 
 [back to top](#readme-top)
 
+## Run ABACUS
+
+After installation, the executable file named `abacus` can be found in the `build` folder. Copy this file and put it in the same folder as all required input files.
+
+Perform calculation by:
+```bash
+mpirun -np 4 abacus
+```
+
+You can replace `4` with your desired number of process, typically the result of the command `nproc`.
+
+Alternatively, if you want to run ABACUS as root, you can add the `--allow-run-as-root` option to the cmd line
+
+```bash
+mpirun -np 4 --allow-run-as-root abacus
+```
```

Comment:
This is the MPI part, which is not the primary concern of ABACUS. I suggest removing it.